packages/design-system/package.json

```diff
@@ -33,9 +33,10 @@
     },
     "./web-components": {
       "types": "./dist/react-components/types/web-components/index.d.ts",
-      "import": "./dist/preact-components/esm/web-components/index.js",
-      "require": "./dist/preact-components/cjs/web-components/index.js"
+      "import": "./dist/web-components/bundle/all.js",
+      "require": "./dist/web-components/bundle/all.js"
```

Comment:
Non-blocking: Does this change mean that there is no difference between esm and cjs anymore?

Reply:
Yeah, that's right—for web components